src/main/java/com/endava/internship/collections/StudentList.java

```diff
@@ -1,26 +1,22 @@
 package com.endava.internship.collections;
 
 
-import java.util.Arrays;
-import java.util.Iterator;
-import java.util.List;
-import java.util.ListIterator;
-import java.util.Collection;
+import java.util.*;
 
-public class StudentList implements List<Student> {
+public class StudentList<T> implements List<T> {
 
     private int size;
     private int maxsize;
-    private Student[] temp;
+    private Object[] temp;
```

Comment:
rename plz to a more explicit name like elementArray

Reply:
Done